src/ExperienceGenerator/Parsing/Factories/LandingPageFactory.cs

```diff
@@ -111,7 +111,7 @@ public override void SetValues(SimulationObject target)
                 target.Variables.Add("Site", site.Id);
                 if (!string.IsNullOrEmpty(site.Host))
                 {
-                    target.Variables.Add("Host", "https://" + site.Host.Split('|')[0]);
+                    target.Variables.Add("Host", "http://" + site.Host.Split('|')[0]);
```

Comment:
@scottmulligan 
Please change this to also not be hardcoded to http.
We shouldn't hardcode either http or https